pkg/util/timeutil/now_test.go

```diff
@@ -16,8 +16,17 @@ package timeutil
 
 import (
 	"testing"
+	"time"
 )
 
+// TestNowUTC asserts that Now()'s timezone is UTC.
```

Comment:
What's a little awkward is that the only way for this test to fail is to have some other package futz with the global, but that package would not be linked in when running the tests in this package.

I guess you could try to import the main cli here (and make this package `timeutil_test`) to avoid the problem?

Reply:
Not required as far as I'm concerned. I assume all sorts of higher-level tests would fail as well if what I described did happen.

Reply:
Yeah, I think that's true, but would be easy to confirm by commenting out the `init` and running `testlogic`.